dashboard/test/controllers/pd/professional_learning_landing_controller_test.rb

```diff
@@ -13,7 +13,10 @@ class Pd::ProfessionalLearningLandingControllerTest < ::ActionController::TestCa
       create :pd_enrollment, email: other_teacher.email, workshop: workshop
     end
 
-    create :pd_enrollment, email: @teacher.email, workshop: @csf_workshop
+    @ended_enrollment = create :pd_enrollment, email: @teacher.email, workshop: @csf_workshop, survey_sent_at: Date.today + 5.days
+    other_enrollment = create :pd_enrollment, email: @teacher.email, workshop: @csd_workshop, survey_sent_at: Date.today + 2.days
```

Comment:
This other_enrollment is not in a normal state with the workshop not ended but a survey sent. It may not affect the test, but it would be cleaner and more representative if the `csd_workshop` was also a `pd_ended_workshop`